### PR TITLE
PgSQL driver & security

### PR DESCRIPTION
**Pull request by JWo1F**

### CODE REVIEW

File: lib/impress.security.pgsql.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+      application.log.warning('No security database configured or can not connect to db');
+      application.security.hasDbWarned = true;
+    }
+    return result;
+  };
+
+  // Create collections and indexes for security subsystem in PostgreSQL
+  //
+  application.security.createDataStructures = function(callback) {
+    console.log('Impress'.green.bold + ' installing initial data structures to PostgreSQL...'.green);
```
Comment (JWo1F) [inline lib/impress.security.pgsql.js]: @tshemsedinov should I use `application.log.debug` instead of `console.log`?

Comment (tshemsedinov) [inline lib/impress.security.pgsql.js]: For debugging yes, but in production code we do not need this output at all

Comment (JWo1F) [inline lib/impress.security.pgsql.js]: @tshemsedinov fixed

**Comment by tshemsedinov** [comment]
https://github.com/metarhia/Impress/issues/626

**Comment by tshemsedinov** [comment]
Sorry, we are going to deprecate security provider, but you can refactor your code and pull PgSQL GlobalStorage provider @JWo1F

**Pull request closed without merging**